samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java

```diff
@@ -0,0 +1,158 @@
+
+        // Category: Normal
+        TStream<String> normalAlerts = normal
+                .filter(tuple -> tuple.get("Systolic") > 80 && tuple.get("Diastolic") > 50)
+                .tag("normal")
+                .map(tuple -> {
+                    return "All is normal. BP is " + tuple.get("Systolic") + "/" +
+                            tuple.get("Diastolic") + ".\n"; })
+                .tag("normal");
+
+        // Category: Prehypertension category
+        TStream<String> prehypertensionAlerts = prehypertension
+                .map(tuple -> {
+                    return "At high risk for developing hypertension. BP is " +
+                            tuple.get("Systolic") + "/" + tuple.get("Diastolic") + ".\n"; })
+                .tag("prehypertension");
+
+        // Category: High Blood Pressure (Hypertension) Stage 1
+        TStream<String> hypertension_stage1Alerts = hypertension_stage1
+                .map(tuple -> {
+                    return "Monitor closely, patient has high blood pressure. " +
+                           "BP is " + tuple.get("Systolic") + "/" + tuple.get("Diastolic") + ".\n"; })
+                .tag("hypertension_stage1")
+                .modify(tuple -> "High Blood Pressure (Hypertension) Stage 1\n" + tuple)
+                .tag("hypertension_stage1");
+
+        // Category: High Blood Pressure (Hypertension) Stage 2
+        TStream<String> hypertension_stage2Alerts = hypertension_stage2
+                .filter(tuple -> tuple.get("Systolic") >= 170 && tuple.get("Diastolic") >= 105)
+                .tag("hypertension_stage2")
+                .peek(tuple ->
+                    System.out.println("BP: " + tuple.get("Systolic") + "/" + tuple.get("Diastolic")))
+                .map(tuple -> {
+                    return "Warning! Monitor closely, patient is at risk of a hypertensive crisis!\n"; })
+                .tag("hypertension_stage2")
+                .modify(tuple -> "High Blood Pressure (Hypertension) Stage 2\n" + tuple)
+                .tag("hypertension_stage2");
+
+        // Category: Hypertensive Crisis
+        TStream<String> hypertensiveAlerts = hypertensive
+                .filter(tuple -> tuple.get("Systolic") >= 180)
+                .tag("hypertensive")
+                .peek(tuple ->
+                    System.out.println("BP: " + tuple.get("Systolic") + "/" + tuple.get("Diastolic")))
+                .map(tuple -> { return "Emergency! See to patient immediately!\n"; })
+                .tag("hypertensive")
+                .modify(tuple -> tuple.toUpperCase())
+                .tag("hypertensive")
+                .modify(tuple -> "Hypertensive Crisis!!!\n" + tuple)
+                .tag("hypertensive");
+
+        // Union two streams to obtain a single stream containing alerts from the normal and
+        // prehypertension alert streams
+        TStream<String> normalAndPrehypertensionAlerts = normalAlerts.union(prehypertensionAlerts);
```

Comment:
Maybe some comment about that, otherwise a reader might be wondering why it was done this way. Maybe something along the lines of additional processing for these streams could go here.